src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl

```diff
@@ -353,7 +356,7 @@
 
 
   <xsl:template mode="mode-iso19139" priority="2005"
-                match="gmd:linkage1111">
+                match="gmd:linkage1111[$schema = 'iso19139.ca.HNAP']">
```

Comment:
This template is not really used, can be removed. Seem was added some dummy value in the match to disable it (`gmd:linkage1111`), this elemetn is handled in the online resources panel

Reply:
removed